docs/index.md

```diff
@@ -1,11 +1,35 @@
 # Introduction to utPLSQL
 
- utPLSQL is a powerful unit testing library for Oracle PL/SQL.
+utPLSQL is a Unit Testing framework for Oracle PL/SQL.
+The framework follows industry standards and best patterns of modern Unit Testing frameworks like [JUnit](http://junit.org/junit4/) and [RSpec](http://rspec.info/)
 
- TODO: Sales pitch of utPLSQL with links to documentation areas that support discussed features.
+#Three steps
+With just three simple steps you can define and run your unit tests for PLSQL code.
+ 
+1. Install the utPLSQL framework 
+2. Create Unit Tests to cover the code
+3. Run the tests
 
- TODO: Table of Contents / Index of the docs
+Here is how you can simply create tested code, unit tests and execute the tests using SQL Developer
 
+![3_steps](images/3_steps_to_run_utPLSQL.gif)
+
+Check out the sections on [annotations](userguide/annotations.md) and [expectations](userguide/expectations.md) to see how to define your tests  
```

Comment:
There is a bit of a conflict here.
The html doc generated with mkdocs contains all the pages and links as a menu.
I wanted to avoid duplication on the index page with what is defined in mkdocs.yml already.
I just pointed references to the things that are mentioned on the index.
We could however have the contents section (similar to the one in `readme.md` file).
The trouble is the duplication of sources.
Maybe we should generate docs from main project page and the `readme.md` should be the main page?
Anyway. That could be done as a further cleanup.